src/NovelRT/CMakeLists.txt

```diff
@@ -89,7 +89,11 @@ set(SOURCES
 
 add_library(Engine SHARED ${SOURCES})
 
-set_property(TARGET Engine PROPERTY OUTPUT_NAME "NovelRT-$<CONFIG>")
+if(CMAKE_BUILD_TYPE MATCHES RELEASE)
```

Comment:
Likewise, generator expression.

Reply:
This is `set_property(TARGET Engine PROPERTY OUTPUT_NAME "$<IF:$<CONFIG:Release>,NovelRT,NovelRT-$<CONFIG>>")` in the newest commit, odd it doesn't show up as out of date o.O